feeds/npm/npm.go

```diff
@@ -73,22 +53,56 @@ func fetchPackages() ([]*Package, error) {
 	if err != nil {
 		return nil, err
 	}
-	return rssResponse.Packages, nil
+	return rssResponse.PackageEvents, nil
 }
 
-// Gets the package version from the NPM.
-func fetchVersionInformation(packageName string) (string, error) {
-	resp, err := httpClient.Get(fmt.Sprintf("%s/%s", versionURL, packageName))
+// Gets the package version & corresponding created date from NPM. Returns
+// a slice of {}Package.
+func createPackages(pkgTitle string, count int) ([]*Package, error) {
```

Comment:
NIT: create seems incorrect here since we're interacting with an external data source.

Reply:
Fair point, changed it to `fetch` inline with the original naming,